internal/pkg/config/env_defaults.go

```diff
@@ -133,13 +135,14 @@ var defaults []*envLimits
 
 func init() {
 	// Packed Files
-	// internal/pkg/config/defaults/1024_limits.yml
-	// internal/pkg/config/defaults/2048_limits.yml
-	// internal/pkg/config/defaults/4096_limits.yml
-	// internal/pkg/config/defaults/8192_limits.yml
-	// internal/pkg/config/defaults/base_limits.yml
+	// internal/pkg/config/defaults/gt10000_limits.yml
```

Comment:
I'm ok with hardcoding it for now but it would be nice if we could come up with a formula instead mid term.

Reply:
yeah, agreed. i'll have to do some more analysis on how much memory/cpu is used by different configurations of agents.

i'm not certain if it exists in fleet-server, but some kind of circuit breaker code could be helpful if we commonly run into situations where the the server is having some issues/gets swamped. i'm interested in what other folks have to say on the topic, as i'm sure there are some implications around leader election at the least.